slash/css/src/List/ContentItemDuo/ContentItemDuo.stories.ts

```diff
@@ -0,0 +1,189 @@
+  label.textContent = args.label;
+  container.appendChild(label);
+
+  const value = document.createElement("p");
+  value.className = "af-content-item-duo__value";
+  value.textContent = args.value;
+  container.appendChild(value);
+
+  if (args.buttonText) {
+    const buttonContainer = document.createElement("div");
+    buttonContainer.className = "af-content-item-duo__button";
+    const button = document.createElement("button");
+    button.className = "af-btn-client af-btn-client--ghost";
+    button.textContent = args.buttonText;
+    buttonContainer.appendChild(button);
+    container.appendChild(buttonContainer);
+  }
+
+  return container;
+};
+
+const defaultArgs = {
+  isVertical: false,
+  label: "Libellé",
+  value: "Réponse",
+  isShowingDoneIcon: true,
+  isShowingCloseIcon: false,
+  buttonText: "En savoir plus",
+  classModifier: [],
+};
+
+const verticalArgs = {
+  isVertical: true,
+  label:
+    "Le choc de véhicules terrestres à moteur, (voiture, trottinette à moteur…) ou la chute d’appareils aériens (avions, hélicoptères…) avec un propriétaire du véhicule adverse identifié ?",
+  value: "Les incendies, explosions, implosions, fumée et foudre",
+  isShowingDoneIcon: true,
+  isShowingCloseIcon: false,
+  buttonText: "En savoir plus",
+  classModifier: [],
+};
+
+const contentItemDuoListRender = ({ classModifier, ...args }: Args) => {
+  const list = document.createElement("section");
+
+  list.className = `af-card ${["list", "extra-padding", ...classModifier]
+    .filter(Boolean)
+    .map((modifier: string) => ` af-card--${modifier}`)
+    .join("")}`;
+
+  list.tabIndex = 0;
+
+  const ul = document.createElement("ul");
+
+  ul.className = "af-list";
+
+  args.items.forEach((childArgs: Args, index: number) => {
+    const item = document.createElement("li");
+    item.className = "af-list__item";
+
+    const newChildArgs = structuredClone(childArgs);
+
+    if (args.isItemsLarge) {
+      newChildArgs.classModifier = ["large"];
+    }
+
+    item.appendChild(contentItemDuoRender(newChildArgs));
+
+    ul.appendChild(item);
+
+    if (index < args.items.length - 1) {
+      const separator = document.createElement("hr");
+      separator.className = "af-list__separator";
+      ul.appendChild(separator);
+    }
+  });
+
+  list.appendChild(ul);
+
+  return list;
+};
+
+export const Default: StoryObj = {
+  render: contentItemDuoRender,
+  args: defaultArgs,
+  argTypes: {
+    classModifier: {
+      options: ["large"],
+      control: { type: "multi-select" },
+      defaultValue: [],
+    },
+  },
+};
+
+export const Vertical: StoryObj = {
+  render: contentItemDuoRender,
+  args: verticalArgs,
+  argTypes: {
+    classModifier: {
+      options: ["large"],
+      control: { type: "multi-select" },
+      defaultValue: [],
+    },
+  },
+};
+
+export const DefaultContentItemDuoList: StoryObj = {
+  render: contentItemDuoListRender,
+  args: {
+    items: [defaultArgs, defaultArgs, defaultArgs],
```

Comment:
Array(3).fill(defaultArgs)

Reply:
Resolved in https://github.com/AxaFrance/design-system/pull/450/commits/49cba16a64e427994989ffc0018ebf052d833e66